R/XcmsExperiment.R

```diff
@@ -1773,7 +1775,9 @@ setMethod(
     "featureSpectra", "XcmsExperiment",
     function(object, msLevel = 2L, expandRt = 0, expandMz = 0, ppm = 0,
              skipFilled = FALSE, return.type = c("Spectra", "List"),
-             features = character(), ...) {
+             features = character(),
+             featureColumns = c("rtmed", "mzmed"),
+             ...) {
         return.type <- match.arg(return.type)
         if (!hasFeatures(object))
```

Comment:
I would also suggest here to first check that `featureColumns` are indeed available in `featureDefinitions()`